### PR TITLE
Feat: Jump to explorer 

### PR DESCRIPTION
**Pull request by Guaris**
https://konghq.atlassian.net/browse/DOCU-4179

### CODE REVIEW

File: app/konnect/analytics/dashboard.md [hunk 1]
```diff
@@ -20,4 +20,14 @@ You can monitor the latency, investigate where delays are noticed, and optimize
...
-You can interact with the traffic and latency graphs, including hovering over chart items to display more details, and selecting a time period using the **time period** drop-down menu. The intervals aggregate data at different increments of time.
\ No newline at end of file
+You can interact with the traffic and latency graphs, including hovering over chart items to display more details, and selecting a time period using the **time period** drop-down menu. The intervals aggregate data at different increments of time. You can also click through a dashboard to navigate to the [explorer](/konnect/analytics/explorer/).
+
+
+## Explorer actions
+
+Charts now offer a new option that enables users to open the chart's content directly in the [explorer](/konnect/analytics/explorer/) view. Clicking on the kebab menu from a chart opens the following actions: 
+
+* **Explore**: This function opens the chart in the [explorer](/konnect/analytics/explorer/) where you can seamlessly drill-down into the data you are seeing from the analytics dashboard.
```
Comment (sichvoge) [inline app/konnect/analytics/dashboard.md:30]:
```suggestion
* **Explore**: This function opens the chart in the [Explorer](/konnect/analytics/explorer/) where you can seamlessly drill-down into the data you are seeing from the analytics dashboard.
```

File: app/assets/images/products/konnect/analytics/konnect-summary-dashboard.png
Comment (sichvoge) [inline app/assets/images/products/konnect/analytics/konnect-summary-dashboard.png:1]: Is that an image from Helen? There is no hover over effect, every chart should display the kebab menu.

**Review by sichvoge — CHANGES_REQUESTED** [review]

File: app/assets/images/products/konnect/analytics/konnect-summary-dashboard.png
Comment (Guaris) [reply app/assets/images/products/konnect/analytics/konnect-summary-dashboard.png:1]: yes

File: app/konnect/analytics/dashboard.md (hunk 1 above)
Comment (Guaris) [reply app/konnect/analytics/dashboard.md:30]: done

**Review by lmilan — APPROVED** [review]

**Pull request merged**